sort/oddeven.go

```diff
@@ -0,0 +1,30 @@
+package sort
+
+//Odd-Even Sort is a variation of bubble sort where the sorting is divided into
+//two phases, Odd and Even Phase and it runs until all the elements are sorted
+//In the odd phase we perform bubble sort on odd indexed elements and in the
+//even phase we perform bubble sort on even indexed elements.
+
```

Comment:
Remove this blank line. GoDoc will not be able to connect to comment with the function.